Classes/Backend/DynamicFlexForm.php

```diff
@@ -162,6 +162,10 @@ public function parseDataStructureByIdentifierPreProcess(array $identifier)
         } else {
             $dataSourceIdentity = $identifier['tableName'] . '.' . $identifier['fieldName'] . ':' . $identifier['uid'];
         }
+        if(!isset(static::$generatedDataSources[$dataSourceIdentity])) {
```

Comment:
This is a screenshot with the current state:
![screenshot 2017-02-01 14 42 06](https://cloud.githubusercontent.com/assets/1185872/22509048/ba1132fe-e88c-11e6-8a75-bb2d11105f87.png)

This is a AJAx request that is being made when a new FAL image is added.
( Have a look at the call stack in the lower left corner )
The identifier was generated by ``FormInlineAjaxController::getFlexFormDataStructureIdentifierFromAjaxContext``.

I am not sure how to change it.
Can you give me a direction?